### PR TITLE
Merge rampup into berkeley

### PR DESCRIPTION
**Pull request by joaosreis**
Merge Rosetta fixes from rampup into berkeley.

### CODE REVIEW

**Comment by joaosreis** [comment]
!ci-build-me

**Comment by joaosreis** [comment]
!ci-nightly-me

**Comment by joaosreis** [comment]
https://buildkite.com/o-1-labs-2/mina-end-to-end-nightlies/builds/1213

**Review by mrmr1993 — APPROVED** [review]

**Comment by mrmr1993** [comment]
!approved-for-mainnet

File: src/app/rosetta/rosetta-cli-config/README.md [hunk 1]
```diff
@@ -116,12 +116,12 @@ is specified in the `config.json`, in `data` section (see the
...
-**IMPORTANT** as of version 0.8.2 of `rosetta-sdk-go`, rosetta-cli
+**IMPORTANT** as of version 0.8.3 of `rosetta-sdk-go`, rosetta-cli
 is unable to run the `mina.ros` file, which will only work with
 [Mina Foundation's fork](https://github.com/MinaProtocol/rosetta-sdk-go/tree/pallas_signer_stake_delegation).
 Therefore, whenever it is required to run the Construction API
-tests against v.0.8.2 of `rosetta-sdk-go`, `mina-no-delegation-tests.ros`
+tests against v.0.8.3 of `rosetta-sdk-go`, `mina-no-delegation-tests.ros`
 config should be used for that instead of `mina.ros`, which is used
-in the CI. `mina-no-delegation-tests.ros` should be deleted once
+in the CI. In alternative, one can also use [our `rosetta-cli` fork](https://github.com/MinaProtocol/rosetta-cli) which is built against our fork of `rosetta-sdk-go`. `mina-no-delegation-tests.ros` should be deleted once
```
Comment (Sventimir) [inline src/app/rosetta/rosetta-cli-config/README.md:125]: This sounds a little better to me.
```suggestion
in the CI. Alternatively, one can also use [our `rosetta-cli` fork](https://github.com/MinaProtocol/rosetta-cli) which is built against our fork of `rosetta-sdk-go`. `mina-no-delegation-tests.ros` should be deleted once
```

**Review by Sventimir — APPROVED** [review]
Looks good to me!

**Comment by joaosreis** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
!approved-for-mainnet

**Review by ejMina226 — APPROVED** [review]

**Pull request merged**